### PR TITLE
Fixed all the errors

### PR DESCRIPTION
**Pull request by Lusindiso**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test.md
+/.github/workflows/linters.yml
```
Comment (sja-thedude) [inline .gitignore]: - [ ] kindly replace this line with node_modules as per requirements.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Lusindiso, This is @sja-thedude your code reviewer.

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required

- [ ] Kindly add all the linter files (.github/workflows , .hintrc , .stylintrc) from [here](https://github.com/microverseinc/linters-config/tree/master/html-css). So, the linters will be set up for your project.
- [ ]  Make sure to add a descriptive Pull Request title and description.
- [ ] _Check the comments under the review._

## Highlights
- Good commit messages 👍 
- Professional Readme 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Lusindiso) [reply .gitignore]: Which line am I suppose to replace?

Comment (sja-thedude) [reply .gitignore]: @Lusindiso , I am talking about Line 2 .

Comment (Lusindiso) [reply .gitignore]: So I am supposed to remove it completely and add /node_modules?

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Lusindiso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

## Things to consider:
- [ ] Please make sure to give your pull request a descriptive title like "Sett up project" or "Add home page", which would make your PR look more professional.
- [ ] Kindly be sure to have a meaningful description for your pull request. This is required later in the program and could invalidate your project, so please make sure you have a good description that says what features you implemented in the PR. 
- [ ] Please make sure to have all linters run successfully and all pass with a green checkmark as this could invalidate your submission.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**